CHANGELOG.md

```diff
@@ -7,25 +7,27 @@ Versions are `MAJOR.PATCH`.
 
 # Changelog
 
-Salt 3003.1 (2021-05-21)
+Salt 3003.1 (2021-06-08)
 ========================
 
 Removed
 -------
 
 - Removed support for Ubuntu 16.04 (#59913)
 
-
 Fixed
 -----
 
+- Fixed race condition in batch logic. Added `listen` option to `LocalClient` to prevent event subscriber from purging cached events during batch iteration. (#56273)
+- Fixed dependencies for Amazon Linux 2 on https://repo.saltproject.io since Amazon Linux 2 now provides some of the python libraries in their repos. (#59982)
+- IPCMessageSubscriber objects expose their connect method as a coroutine so they
```

Comment:
just a format issue. no newline, as the previous lines do not have newlines either